src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SuggestionModeCompletionProviderTests.cs

```diff
@@ -391,7 +391,7 @@ static void Main(string[] args)
         Console.CancelKeyPress += new ConsoleCancelEventHandler(((a$$
     }
 }";
-            await VerifyBuilderAsync(markup);
+            await VerifyNotBuilderAsync(markup);
```

Comment:
Traced this back to this [line/commit](https://github.com/dotnet/roslyn/commit/a3d8ab1b34eab45abf56e83178d3ddbe66a5c49f#diff-532cf6fcbe8d404c428ba6de669fd3b7R394) which incorrectly modified the test.

`Console.CancelKeyPress += new ConsoleCancelEventHandler((a$$` in the test above shows a builder.
But `Console.CancelKeyPress += new ConsoleCancelEventHandler(((a$$` should not.
This was hidden for a while by the introduction of a builder when in possible tuple context.